app/services/windows.ts

```diff
@@ -203,6 +203,20 @@ export class WindowsService extends StatefulService<IWindowsState> {
     // This prevents "snapping" behavior when navigating settings
     if (options.componentName !== this.state.child.componentName) options.center = true;
 
+    // Override `options.size` when what is passed in is bigger than the current display
+    if (options.size) {
+      const { screen } = require('electron');
```

Comment:
This file already has `electron` imported at the top of the file.

Reply:
We can't import `screen` like that since it's apparently reserved. Run into this myself.

Reply:
Access it as `electron.screen`

Reply:
It's accessed this way elsewhere in the file